### PR TITLE
Add OXXO Payment Method

### PR DESCRIPTION
**Pull request by pololi-stripe**
## Summary
Add OXXO Payment Method

## Motivation
To support OXXO payment method in iOS SDK.

## Testing
- Functional test `STPPaymentMethodOXXOParamsTests` and  `STPPaymentMethodOXXOTests `
  - Note: since the OXXO requires beta header, I need to override [`STPAPIClient APIVersion`](https://github.com/stripe/stripe-ios/blob/master/Stripe/STPAPIClient.m#L57)locally to run the test 
- Non-card example
Xnip2020-06-23_23-12-27
Xnip2020-06-23_23-12-04
Xnip2020-06-23_23-12-12
Xnip2020-06-23_23-12-20

### CODE REVIEW

**Comment by pololi-stripe** [comment]
The test failed on CI because OXXO related tests requires `beta` header. I've ran those tests locally with override beta version.

File: Stripe/STPPaymentMethodOXXO.m [hunk 1]
```diff
@@ -0,0 +1,50 @@
+//
+//  STPPaymentMethodOXXO.m
```
Comment (yuki-stripe) [inline Stripe/STPPaymentMethodOXXO.m:2]: Make sure these `.m` files are also in the StripeiOSStatic target

File: Stripe/PublicHeaders/STPIntentActionDisplayOXXODetails.h [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#import <Foundation/Foundation.h>
+
+#import "STPAPIResponseDecodable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ Contains OXXO details necessary for the customer to complete the payment.
+ */
+
```
Comment (yuki-stripe) [inline Stripe/PublicHeaders/STPIntentActionDisplayOXXODetails.h:18]: nit delete this space, might mess with doc autogen

Comment (pololi-stripe) [reply Stripe/PublicHeaders/STPIntentActionDisplayOXXODetails.h]: fixed in https://github.com/stripe/stripe-ios/pull/1592/commits/d1119ead20214924cf571135071f9869cfc0d41b

File: Stripe/STPPaymentMethodOXXO.m (hunk 1 above)
Comment (pololi-stripe) [reply Stripe/STPPaymentMethodOXXO.m:2]: Fixed in https://github.com/stripe/stripe-ios/pull/1592/commits/236a98dad6d03f34b03e76ed877799e479e2bd45

**Comment by davidme-stripe** [comment]
I merged this to the latest, cleaned up the `next_action` changes since the beta, and tested against Non-Card Payment Examples. @csabol-stripe or @yuki-stripe, can you give it another review?

**Review by yuki-stripe — APPROVED** [review]

**Comment by pololi-stripe** [comment]
💯 Thank you ❤️ !
I'll assign back to you to merge it. Thank you! @davidme-stripe

**Comment by carlosdelamora** [comment]
I am curious about how the flow will work. Is there a place where you are writing about it?

**Comment by davidme-stripe** [comment]
Hey @carlosdelamora! We've added some docs for using OXXO on iOS: https://stripe.com/docs/payments/oxxo/accept-a-payment#ios

**Pull request merged**